src/renderer/views/wallet/NoWalletView/NoWalletView.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useMemo } from 'react'
+
+import Tabs from '../../../components/Tabs'
+import { MainnetView } from './MainnetView'
+
+enum TabKey {
+  TESTNET = 'testnet',
+  MAINNET = 'mainnet'
+}
+
+type Tab = {
+  key: TabKey
+  label: string
+  content: React.ReactNode
+}
+
+const NoWalletView = () => {
+  const items: Tab[] = useMemo(() => [{ key: TabKey.MAINNET, label: 'mainnet', content: <MainnetView /> }], [])
+
+  return <Tabs centerContent={true} tabs={items} />
+}
+
+export default NoWalletView
```

Comment:
Since we don't need any tabs here, I would recommend to replace all of current content with all content of `MainnetView`. Layout has been updated https://www.figma.com/file/LgRqZCGMShZ2mJ702MIqyV/ASGARDEX-2020?node-id=1310%3A0 . `MainnetView` can be removed then. This change can be done by another PR ...